frontend/src/components/content-types/ContentTypeDialog.tsx

```diff
@@ -112,12 +112,41 @@ export const ContentTypeDialog: FC<ContentTypeDialogProps> = ({
               <Input
                 label={t("label.name")}
                 error={!!errors.name}
-                {...register("name", validationRules.name)}
-                helperText={errors.name ? errors.name.message : null}
+                {...register("name", {
+                  required: t("message.name_is_required"),
+                })}
```

Comment:
Hi @IkbelTalebHssan, i have noticed that the input **helperText** attribute is accidentally removed

```suggestion
                {...register("name", {
                  required: t("message.name_is_required"),
                })}
                helperText={errors.name ? errors.name.message : null}
```

Reply:
> accidentally removed

THank you @yassinedorbozgithub for the review :pray 
Fixed :heavy_check_mark: